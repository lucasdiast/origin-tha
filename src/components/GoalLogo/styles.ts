import styled from 'styled-components';

export const Container = styled.div`
  display: flex;
  justify-content: flex-start;
  align-items: center;
  margin-bottom: 32px;
`;

export const Logo = styled.img`
  margin-right: 16px;
`;

export const GoalTitle = styled.span`
  display: block;
  font-size: 24px;
  margin-bottom: 8px;
  font-family: 'Rubik';
  font-weight: 500;

  @media (max-width: 560px) {
    font-size: 20px;
  }
`;
