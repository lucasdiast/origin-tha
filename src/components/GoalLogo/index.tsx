import React, { ReactElement } from 'react';
import { Container, GoalTitle, Logo } from './styles';

interface AuxProps {
  goal: string;
  logo: string;
}

const GoalLogo = ({ goal, logo }: AuxProps): ReactElement => {
  return (
    <Container>
      <Logo src={logo} alt="goal" />
      <div>
        <GoalTitle>{goal}</GoalTitle>
        <span>Saving goal</span>
      </div>
    </Container>
  );
};

export default GoalLogo;
