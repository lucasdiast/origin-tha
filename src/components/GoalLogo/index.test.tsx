import { shallow } from 'enzyme';
import GoalLogo from './index';
import logo from '../../assets/icons/logo.svg';

describe('GoalLogo', () => {
  it('renders correctly', () => {
    const component = shallow(
      <GoalLogo goal={'Text that should appear'} logo={logo} />
    );

    expect(component).toMatchSnapshot();
  });
});
