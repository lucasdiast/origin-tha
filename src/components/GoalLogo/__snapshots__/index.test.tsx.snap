// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoalLogo renders correctly 1`] = `ShallowWrapper {}`;
